app/src/main/res/drawable/rounded_rect_grey_border_gradient_white_background.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+  <stroke
+    android:width="1dp"
+    android:color="@color/oppiaSubmittedAnswerTextColor" />
+  <gradient
+    android:angle="270"
+    android:centerColor="#EFEFEF"
+    android:endColor="#DCDCDC"
+    android:startColor="#FEFEFE"
```

Comment:
Nit suggested:
```
    android:centerColor="#F7F7F7"
    android:endColor="#DCDCDC"
    android:startColor="#FFFFFF"
```

Suggested `F7` because it is the color in between `FE` and `EF`

Reply:
done